app/validators/questionnaire_validator.py

```diff
@@ -104,3 +104,22 @@ def validate_smart_quotes(self):
                         error_messages.DUMB_QUOTES_FOUND,
                         pointer=translatable_item.pointer,
                     )
+
+    def validate_white_spaces(self):
```

Comment:
This doesn't seem to be called anywhere, so the validation won't work yet?.

Reply:
Code refactored so that function will be in the QuestionnaireValidator class